examples/pyfunc/src/train.py

```diff
@@ -0,0 +1,58 @@
+from mlflow.tracking import MlflowClient
+from sklearn.datasets import load_iris
+from sklearn.linear_model import LogisticRegression
+
+import mlflow
+import mlflow.sklearn
+
+
+class CustomPredict(mlflow.pyfunc.PythonModel):
+    """Custom pyfunc class used to create customized mlflow models"""
+
+    def load_context(self, context):
+
+        self.model = mlflow.sklearn.load_model(context.artifacts["custom_model"])
+
+    def predict(self, context, model_input):
+
+        return self.model.predict(model_input)
+
+
+X, y = load_iris(return_X_y=True, as_frame=True)
+params = {"C": 1.0, "random_state": 42}
+
+# create an mlflow client
+client = MlflowClient()
+
+model_name = "base_model"
+custom_model_name = "custom_model"
+
+with mlflow.start_run(run_name="test_pyfunc") as train_run:
+
+    regression_model = LogisticRegression(**params).fit(X, y)
+
+    model_info = mlflow.sklearn.log_model(
+        sk_model=regression_model, artifact_path="model", registered_model_name=model_name
+    )
+
+    # start a child run to create custom imagine model
+    with mlflow.start_run(run_name="test_custom_model", nested=True) as run:
+
+        # log a custom model
+        mlflow.pyfunc.log_model(
+            artifact_path="",
```

Comment:
```suggestion
            artifact_path="model",
```

Can we log the model artifacts in `model`?

Reply:
Using sqlite as `backend-store-uri` works fine. But test suite fails as it seems that the backend is `file` based.
I can see two options here:
 - To omit `registered_model_name=` and model transitioning
 - To find a workaround to enable sqlite backend for this particular test in test suite runtime.

Am I loosing something ?

Reply:
Model registry doesn't support file backend :)

Reply:
Let's remove model registry operations

Reply:
yeap, i agree.
Then, I will remove model transitioning in order to make this work